website/docs/.vuepress/config.js

```diff
@@ -128,7 +128,8 @@ module.exports = {
               title: "Illumina Genotyping Arrays Pipeline",
               collapsable: true,
               children: [
-                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation"
+                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/IlluminaGenotypingArray.documentation",
+                "Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec"
```

Comment:
I think I'd call it Illumina_genotyping_array_spec, just to be consistent.

Reply:
Updated this!